doc/api/child_process.md

```diff
@@ -113,6 +113,9 @@ exec('my.bat', (err, stdout, stderr) => {
 ```
 
 ### child_process.exec(command[, options][, callback])
+<!-- YAML
+added: v0.1.33
+-->
```

Comment:
04c06b91493762dea34d91c34f2a33ebaa919eca uses `require("child_process").exec` in the docs, that’s pretty much what I went by here. But yeah, sorry, this is one version off.